nvflare/fuel/common/ctx.py

```diff
@@ -41,11 +38,8 @@ def clear_props(self):
 
 
 class BaseContext(SimpleContext):
-    """
-    A SimpleContext with threading locks.
-    """
-
     def __init__(self):
+        """A SimpleContext with threading locks."""
```

Comment:
we might want to improve this.
What does this context serve? Why it needs to have asked_to_stop parameter

Reply:
This context is only used in low level library and actually ask_to_stop() is never called.  We can remove this parameter.